mediapipe/tasks/ios/core/sources/MPPExternalFile.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2022 The MediaPipe Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Holds information about an external file.
+ */
+NS_SWIFT_NAME(ExternalFile)
+@interface MPPExternalFile : NSObject <NSCopying>
```

Comment:
Note that this is inlined in the C++/Java BaseOptions for MediaPipe and not exposed as an external class.

Reply:
Instead of having properties for path, byte buffer, file descriptor etc., in MPPBaseOptions, a single class encapsulating all file input methods seemed a better way. Idea is that any new file input method will become a new property in this class Right now, we are only handling path to tflite file. But others will be added in future. 
LMK if having them directly in base options is better.

Reply:
We should follow the C++ API as much as possible. As such, I have inlined this class in the internal version of this PR. You will see the update once I obtained approval. Thanks!